configuration/fismarker.properties

```diff
@@ -0,0 +1 @@
+2.18.1.redhat-000012
```

Comment:
yes there is a abom
<groupId>io.fabric8</groupId>
        <artifactId>fabric8-project-bom-camel-spring-boot</artifactId>
        <version>${fabric8.version}</version>